### ISSUE TITLE
RuntimeError (There's no setting named plugin_lock_out)

### ISSUE THREAD

**Issue by paulbutenko** [omegacodepl/redmine_lock_out#1]
Hello,

I have tried to download and setup this plugin.
Installation looks correct - I see Lock out dates, Plugin configuration, etc

plugin installed in folder `redmine_lock_out`
redmine version 4.1.1

however when I try to add new Spend time entry for current date or edit another one for unlocked and current month, I'm getting error 500
![image](https://user-images.githubusercontent.com/5246804/115989196-8e50e400-a5bd-11eb-8465-9e8963f0473e.png)

and log message

```
Started POST "/time_entries" for 172.20.0.4 at 2021-04-25 09:56:30 +0000
Processing by TimelogController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"fD8wYaPVglKQNAlJXByKrOQmVCHbt+U31rkKFB5ECwAwYyDMzj0Sw2DF1AxAHFEtYeV1NwOM3ZLmExGliyubug==", "back_url"=>"/time_entries", "time_entry"=>{"project_id"=>"1", "issue_id"=>"10", "spent_on"=>"2021-04-25", "hours"=>"1", "comments"=>"", "activity_id"=>"5"}, "commit"=>"Create"}
  Current user: pavlo.butenko (id=14)
Completed 500 Internal Server Error in 22ms (ActiveRecord: 9.2ms)

RuntimeError (There's no setting named plugin_lock_out):

app/models/setting.rb:330:in `find_or_default'
app/models/setting.rb:117:in `[]'
plugins/redmine_lock_out/lib/lock_out/time_entry_patch.rb:26:in `lock_out_date'
plugins/redmine_lock_out/lib/lock_out/time_entry_patch.rb:22:in `spent_on_valid?'
plugins/redmine_lock_out/lib/lock_out/time_entry_patch.rb:15:in `check_date_for_lock_out'
app/controllers/timelog_controller.rb:109:in `create'
lib/redmine/sudo_mode.rb:65:in `sudo_mode'
```

any idea what could be the problem?

Thank you

**Comment by omegacodepl** [comment]
Hi
Your problem was existing because plugin settings were used wrong way. This caused to fetch nil instead of default value for lock out day setting. Problem is resolved in 2.0.3

**Comment by paulbutenko** [comment]
Hello,
Thank you, fix is working.
